opencog/nlp/irc/IRC.cc

```diff
@@ -149,7 +149,7 @@ int IRC::start(const char* server, int port, const char* nick,
 
 	/* Ping every .. I dunno -- 5 minutes? */
 	optval = 300;
-	rc=setsockopt(irc_socket, IPPROTO_TCP, TCP_KEEPIDLE, &optval, optlen);
+	rc=setsockopt(irc_socket, IPPROTO_TCP, TCP_KEEPCNT, &optval, optlen);
 	if (0 > rc)

```

Comment:
Hmm. Actually, both the original, and your change are wrong; it should have been `TCP_KEEPINTVL` -- does MacOS support that?

Reply:
Yes it does, thanks.